config.go

```diff
@@ -22,6 +22,7 @@ type exportedTagsOnMetrics map[string][]string
 
 type job struct {
 	Region        string   `yaml:"region"`
```

Comment:
There seems to be still region in there. Can we remove this?

Reply:
i'll remove it.  kept it in so it wouldnt be a breaking change, just new functionality - at least for a few versions.